proposals/simd/simdsignandunsignedextendedloads.md

```diff
@@ -0,0 +1,64 @@
+### **Proposal WebAssembly SIMD Modification**
+
+Currently as proposed there is an instructions defined in the WASM SIMD ISA as follows.
+
+**i8x16.mul** which is a register to register operation that takes 16 8 bit integers and
+
+multiplies them together resulting in an 8 bit value. If the distribution of the integers it flat this
+
+would result in a large percent of the instructions with overflow. This is a problem for many applications.
```

Comment:
i8x16.mul implements wrapping integer multiplication, therefore, I don't see why overflow there is a problem (it's not an error, nor UB, but perfectly defined)